sample_apps_reinvent2021/go/README.md

```diff
@@ -0,0 +1,71 @@
+# Getting started with Amazon Timestream with Go
+This sample application shows to
+1. Create table enabled with magnetic tier upsert
+2. Ingest data with multi measure records
+3. Create ScheduledQuery and interpret its run.
+
+This application populates the table with ~63K rows of sample  multi measure value data (provided as part of csv) , and run sample queries to jumpstart your evaluation and/or proof-of-concept applications with Amazon Timestream.
+
+Run the command `chmod 777 *.sh` on the .sh files in go_sample to make them executable files
+
+#### Sample application to create scheduled query and run
+- View list of arguments
+  ```
+  go run scheduled-query-sample.go --help
+   ```
+- Run example to create scheduled query and run
+   ```
+  go run scheduled-query-sample.go 
+   ```
+- Run example to ingest the multi measure data in `sample-multi.csv`. DEFAULT value is `../data/sample-multi.csv`
+  ```
+  go run scheduled-query-sample.go --csv_file_path="../data/sample-multi.csv"
+  ```
+
+- Run example to create scheduled query that fails on execution and generates an error report. DEFAULT value is false,
+  so it runs a valid query in the scheduled query sample
+  ```
+  go run scheduled-query-sample.go --run_invalid_scheduled_query_flag=true
+  ```
+- Delete resources created by this script for creating/executing E2E scheduled query. DEFAULT value is true
+  ```
+  go run scheduled-query-sample.go --delete_resources_after_execution_flag=false
```

Comment:
should the value be true or false. If false, fix the description

Reply:
we have set the default as true in the src.   also given we have option to pass a flag we should be good with any default. thoughts ?

Reply:
The comment says delete resources but flag value is false.